xsd/sdp-manifest.xsd

```diff
@@ -60,4 +61,47 @@
 		</xsd:attribute>
 	</xsd:complexType>
 
+    <xsd:complexType name="Lenke">
+        <xsd:sequence>
+            <xsd:element name="url" type="HttpLenke" minOccurs="1" maxOccurs="1"/>
+            <xsd:element name="beskrivelse" type="LenkeBeskrivelseTekst" minOccurs="0" maxOccurs="1"/>
+            <xsd:element name="knappTekst" type="LenkeKnappTekst" minOccurs="0" maxOccurs="1"/>
+            <xsd:element name="frist" type="xsd:dateTime" minOccurs="0" maxOccurs="1"/>
+        </xsd:sequence>
+    </xsd:complexType>
+
+    <xsd:simpleType name="HttpLenke">
+        <xsd:restriction base="xsd:anyURI">
+            <xsd:pattern value="https?://.+" />
```

Comment:
Jeg fjernet valideringen i digipost-data-types, men kan jo ha en som er tilsvarende denne.

Reply:
Ja, dette er jo en veldig enkel og grei sjekk som virker hensiktsmessig. _Kanskje_ vi burde slengt på en `^` i starten, nå garanterer vi jo for så vidt ikke at det er en gyldig lenke.

Reply:
Ja `^` i starten er bra

Reply:
Uh, stryk det. 

> XML schema always implicitly anchors the entire regular expression.

https://www.regular-expressions.info/xml.html

TL;DR `^` (og `$`) er implisitt inkludert i regex-er i XSDer 🤷‍♂️